### ISSUE TITLE
Swap <a /> with <button /> for <SortableTable />

### ISSUE THREAD

**Issue by kelsonic** [department-of-veterans-affairs/va.gov-team#4558]
# Description
The `<SortableTable />` component from the [`formation-react`](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/sortabletable/) library has a potential accessibility issue. Its table header cells are anchor tags instead of buttons, despite not utilizing an `href`.

## Tasks
- [x] Increment package JSON version in formation and VETS website - .5 hours (Kelson)
- [x] Swap a tag with button type - .5 hours
- [x] Update formation css to make sure the button looks the same as the a tag did. - .5 hours

## Screenshots
![image.png](https://images.zenhubusercontent.com/5d812458c75aaf0001c430f8/e6acc3e4-42bd-4463-af3f-276a1eaee7f6)

## Acceptance Criteria
- [x] Column title cells are buttons instead of anchor tags.

**Comment by ncksllvn** [comment]
@kelsonic I think you can simply add the helper class, `va-button-link`, to a button to make it appear as a link. That should make this change easier

**Comment by kelsonic** [comment]
Oh awesome! Thank you!!

**Comment by Karak888** [comment]
everyone should benefit from sortable table!

**Comment by kelsonic** [comment]
It's in staging! 🙂 

![image](https://user-images.githubusercontent.com/12773166/73278317-a7939680-41a8-11ea-9b82-d8a76f01c63c.png)
